core/src/main/java/nu/yona/server/properties/SecurityProperties.java

```diff
@@ -5,8 +5,9 @@
 	private int newDeviceRequestExpirationDays = 1;
 	private int passwordLength = 32;
 	private long bruteForceBlockMinutes = 60;
-	private long dosMemorySeconds = 300;
-	private boolean isDOSProtectionEnabled = false;
+	private long dosProtectionWindowSeconds = 300;
+	private boolean isDosProtectionEnabled = false;
+	private int maxCreateUserAttemptsPerTimeWindow = 1;
```

Comment:
Not necessarily. The two defaults don't need to be identical. I'd rather make the one on the code very very strict, so you know that you're not working with realistic data.